sda/cmd/ingest/ingest.go

```diff
@@ -100,6 +101,24 @@ func main() {
 	var message schema.IngestionTrigger
 
 	go func() {
+		start := time.Now()
+		for i := 0; i <= 10; i++ {
```

Comment:
Maybe better to define a variable or constant instead of hard coding the number of loops.